vpr/src/route/route_timing.cpp

```diff
@@ -159,10 +160,10 @@ static void timing_driven_expand_node_non_configurable_recurr(const float critic
 t_timing_driven_node_costs evaluate_timing_driven_node_costs(const t_timing_driven_node_costs old_costs,
     const float criticality_fac, const float bend_cost, const float astar_fac,
     const route_budgets& budgeting_inf, const float max_delay, const float min_delay, const float target_delay, const float short_path_crit,
-    const int from_node, const int to_node, const int iconn, const int target_node);
+    const int from_node, const int to_node, const int iconn, const int target_node, const float T_upstream);
 
 static float get_timing_driven_expected_cost(int inode, int target_node,
-        float criticality_fac, float R_upstream);
+        float criticality_fac, float R_upstream, int get_delay);
```

Comment:
It looks like the `get_delay` parameter is used only as 0 or 1. It would be better to use a `bool` than an `int` in that case.

There should also be a comment describing what the `get_delay` flag means.